policyengine_us/parameters/gov/states/ks/tax/income/agi/subtractions/oasdi/agi_limit.yaml

```
@@ -0,0 +1,13 @@
+description: KS federal AGI limit to be eligible for taxable OASDI subtraction.
```

Could you please amend descriptions to read like this?
```suggestion
description: Kansas limits its taxable OASDI subtraction to filers with AGI below this amount.
```

---

Done.